js/containers/landingsside/DineOppgaverContainer.js

```diff
@@ -145,9 +146,15 @@ export const mapStateToProps = (state) => {
     const sykepengesoknader = state.sykepengesoknader.data.filter((s) => {
         return s.status === sykepengesoknadstatuser.NY;
     });
-    const soknader = state.soknader.data.filter((s) => {
-        return s.status === NY && s.soknadstype === SELVSTENDIGE_OG_FRILANSERE;
-    });
+    const soknader = state.soknader.data
+        .filter((s) => {
```

Comment:
er det ikke mer effektivt å slå sammen filterne? Ellers må man vel loope gjennom to ganger.

Reply:
Jeg tenker det er ryddig å holde dem adskilt

Reply:
evt wrappe dem i to metoder og ta dem sammen